### ISSUE TITLE
Plan integration error.

### ISSUE THREAD

**Issue by RikoDEV** [DiscordSRV/DiscordSRV#457]
Server: `This server is running Paper version git-Paper-11 (MC: 1.14) (Implementing API version 1.14-R0.1-SNAPSHOT)`
Java: `Java 1.8 (Java HotSpot(TM) 64-Bit Server VM 25.202-b08) Host: Linux 4.9.0-8-amd64 (amd64)`

```
[10:41:52 WARN]: [Plan] DiscordSRV ran into IllegalArgumentException (but failed safely) when updating value for 'server', the method was disabled temporarily (won't be called until next Plan reload), stack trace to follow:
[10:41:52 WARN]: [Plan] Error was caught by com.djrapitops.plan.extension.ExtensionServiceImplementation
java.lang.IllegalArgumentException: class com.djrapitops.extension.DiscordSRVExtension method accountsLinkedPerGuild had invalid parameters; caused java.lang.NullPointerException
at com.djrapitops.plan.extension.implementation.providers.MethodWrapper.callMethod(MethodWrapper.java:85) ~[?:?]
at com.djrapitops.plan.extension.implementation.providers.MethodWrapper.callMethod(MethodWrapper.java:64) ~[?:?]
at com.djrapitops.plan.extension.implementation.providers.gathering.DoubleAndPercentageProviderValueGatherer.lambda$null$4(DoubleAndPercentageProviderValueGatherer.java:82) ~[?:?]
at com.djrapitops.plan.extension.implementation.providers.gathering.DoubleAndPercentageProviderValueGatherer.getMethodResult(DoubleAndPercentageProviderValueGatherer.java:122) ~[?:?]
at com.djrapitops.plan.extension.implementation.providers.gathering.DoubleAndPercentageProviderValueGatherer.gatherDoubleDataOfProvider(DoubleAndPercentageProviderValueGatherer.java:105) ~[?:?]
at com.djrapitops.plan.extension.implementation.providers.gathering.DoubleAndPercentageProviderValueGatherer.gatherDoubleDataOfServer(DoubleAndPercentageProviderValueGatherer.java:87) ~[?:?]
at com.djrapitops.plan.extension.implementation.providers.gathering.ProviderValueGatherer.updateValues(ProviderValueGatherer.java:137) ~[?:?]
at com.djrapitops.plan.extension.ExtensionServiceImplementation.updateServerValues(ExtensionServiceImplementation.java:189) ~[?:?]
at com.djrapitops.plan.extension.ExtensionServiceImplementation.updateServerValues(ExtensionServiceImplementation.java:196) ~[?:?]
at com.djrapitops.plan.extension.ExtensionServiceImplementation.updateServerValues(ExtensionServiceImplementation.java:196) ~[?:?]
at com.djrapitops.plan.extension.ExtensionServiceImplementation.updateServerValues(ExtensionServiceImplementation.java:196) ~[?:?]
at com.djrapitops.plan.extension.ExtensionServiceImplementation.updateServerValues(ExtensionServiceImplementation.java:196) ~[?:?]
at com.djrapitops.plan.extension.ExtensionServiceImplementation.updateServerValues(ExtensionServiceImplementation.java:196) ~[?:?]
at com.djrapitops.plan.extension.ExtensionServiceImplementation.updateServerValues(ExtensionServiceImplementation.java:196) ~[?:?]
at com.djrapitops.plan.extension.ExtensionServiceImplementation.updateServerValues(ExtensionServiceImplementation.java:196) ~[?:?]
at com.djrapitops.plan.extension.ExtensionServiceImplementation.updateServerValues(ExtensionServiceImplementation.java:196) ~[?:?]
at com.djrapitops.plan.extension.ExtensionServiceImplementation.updateServerValues(ExtensionServiceImplementation.java:196) ~[?:?]
at com.djrapitops.plan.extension.ExtensionServiceImplementation.updateServerValues(ExtensionServiceImplementation.java:196) ~[?:?]
at com.djrapitops.plan.extension.ExtensionServiceImplementation.updateServerValues(ExtensionServiceImplementation.java:196) ~[?:?]
at com.djrapitops.plan.extension.ExtensionServiceImplementation.updateServerValues(ExtensionServiceImplementation.java:196) ~[?:?]
at com.djrapitops.plan.extension.ExtensionServiceImplementation.updateServerValues(ExtensionServiceImplementation.java:196) ~[?:?]
```

**Comment by Vankka** [comment]
@RikoDEV Are you able to reproduce the issue?

I talked to Plan's dev about this, if you could please try with this version:
https://www.dropbox.com/s/bi4dh57y033o2mo/Plan%206d788ba.zip?dl=0
and send the errors if you get them

**Comment by RikoDEV** [comment]
@Vankka Now throwing:

```
[19:49:40 WARN]: [Plan] DiscordSRV ran into IllegalArgumentException (but failed safely) when updating value for 'server', the method was disabled temporarily (won't be called until next Plan reload), stack trace to follow:
[19:49:40 WARN]: [Plan] Error was caught by com.djrapitops.plan.extension.ExtensionServiceImplementation
java.lang.IllegalArgumentException: class com.djrapitops.extension.DiscordSRVExtension method accountsLinkedPerGuild could not be called: null
at com.djrapitops.plan.extension.implementation.providers.MethodWrapper.callMethod(MethodWrapper.java:86) ~[?:?]
at com.djrapitops.plan.extension.implementation.providers.MethodWrapper.callMethod(MethodWrapper.java:65) ~[?:?]
at com.djrapitops.plan.extension.implementation.providers.gathering.DoubleAndPercentageProviderValueGatherer.lambda$null$4(DoubleAndPercentageProviderValueGatherer.java:82) ~[?:?]
at com.djrapitops.plan.extension.implementation.providers.gathering.DoubleAndPercentageProviderValueGatherer.getMethodResult(DoubleAndPercentageProviderValueGatherer.java:122) ~[?:?]
at com.djrapitops.plan.extension.implementation.providers.gathering.DoubleAndPercentageProviderValueGatherer.gatherDoubleDataOfProvider(DoubleAndPercentageProviderValueGatherer.java:105) ~[?:?]
at com.djrapitops.plan.extension.implementation.providers.gathering.DoubleAndPercentageProviderValueGatherer.gatherDoubleDataOfServer(DoubleAndPercentageProviderValueGatherer.java:87) ~[?:?]
at com.djrapitops.plan.extension.implementation.providers.gathering.ProviderValueGatherer.updateValues(ProviderValueGatherer.java:137) ~[?:?]
at com.djrapitops.plan.extension.ExtensionServiceImplementation.updateServerValues(ExtensionServiceImplementation.java:189) ~[?:?]
at com.djrapitops.plan.extension.ExtensionServiceImplementation.updateServerValues(ExtensionServiceImplementation.java:196) ~[?:?]
at com.djrapitops.plan.extension.ExtensionServiceImplementation.updateServerValues(ExtensionServiceImplementation.java:196) ~[?:?]
at com.djrapitops.plan.extension.ExtensionServiceImplementation.updateServerValues(ExtensionServiceImplementation.java:196) ~[?:?]
at com.djrapitops.plan.extension.ExtensionServiceImplementation.updateServerValues(ExtensionServiceImplementation.java:196) ~[?:?]
at com.djrapitops.plan.extension.ExtensionServiceImplementation.lambda$register$0(ExtensionServiceImplementation.java:102) ~[?:?]
at com.djrapitops.plan.system.processing.Processing.lambda$submitNonCritical$0(Processing.java:73) ~[?:?]
at java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1590) ~[?:1.8.0_202]
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:1.8.0_202]
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:1.8.0_202]
at java.lang.Thread.run(Thread.java:748) [?:1.8.0_202]
Caused by: java.lang.reflect.InvocationTargetException
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_202]
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_202]
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_202]
at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_202]
at com.djrapitops.plan.extension.implementation.providers.MethodWrapper.callMethod(MethodWrapper.java:72) ~[?:?]
... 17 more
Caused by: java.lang.NullPointerException
at com.djrapitops.extension.DiscordSRVExtension.getLinkedAccountCount(DiscordSRVExtension.java:187) ~[?:?]
at com.djrapitops.extension.DiscordSRVExtension.accountsLinkedPerGuild(DiscordSRVExtension.java:217) ~[?:?]
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_202]
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_202]
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_202]
at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_202]
at com.djrapitops.plan.extension.implementation.providers.MethodWrapper.callMethod(MethodWrapper.java:72) ~[?:?]
... 17 more
[19:49:40 INFO]: [CrazyAuctions] Enabling CrazyAuctions v1.2.7
[19:49:40 INFO]: [CrazyAuctions] Loading the config.yml
[19:49:40 INFO]: [CrazyAuctions] Successfully loaded config.yml
[19:49:40 INFO]: [CrazyAuctions] Loading the Data.yml
[19:49:40 INFO]: [CrazyAuctions] Successfully loaded Data.yml
[19:49:40 WARN]: [Plan] DiscordSRV ran into IllegalArgumentException (but failed safely) when updating value for 'server', the method was disabled temporarily (won't be called until next Plan reload), stack trace to follow:
[19:49:40 WARN]: [Plan] Error was caught by com.djrapitops.plan.extension.ExtensionServiceImplementation
java.lang.IllegalArgumentException: class com.djrapitops.extension.DiscordSRVExtension method accountsLinkedPerGuild could not be called: null
at com.djrapitops.plan.extension.implementation.providers.MethodWrapper.callMethod(MethodWrapper.java:86) ~[?:?]
at com.djrapitops.plan.extension.implementation.providers.MethodWrapper.callMethod(MethodWrapper.java:65) ~[?:?]
at com.djrapitops.plan.extension.implementation.providers.gathering.DoubleAndPercentageProviderValueGatherer.lambda$null$4(DoubleAndPercentageProviderValueGatherer.java:82) ~[?:?]
at com.djrapitops.plan.extension.implementation.providers.gathering.DoubleAndPercentageProviderValueGatherer.getMethodResult(DoubleAndPercentageProviderValueGatherer.java:122) ~[?:?]
at com.djrapitops.plan.extension.implementation.providers.gathering.DoubleAndPercentageProviderValueGatherer.gatherDoubleDataOfProvider(DoubleAndPercentageProviderValueGatherer.java:105) ~[?:?]
at com.djrapitops.plan.extension.implementation.providers.gathering.DoubleAndPercentageProviderValueGatherer.gatherDoubleDataOfServer(DoubleAndPercentageProviderValueGatherer.java:87) ~[?:?]
at com.djrapitops.plan.extension.implementation.providers.gathering.ProviderValueGatherer.updateValues(ProviderValueGatherer.java:137) ~[?:?]
at com.djrapitops.plan.extension.ExtensionServiceImplementation.updateServerValues(ExtensionServiceImplementation.java:189) ~[?:?]
at com.djrapitops.plan.extension.ExtensionServiceImplementation.updateServerValues(ExtensionServiceImplementation.java:196) ~[?:?]
at com.djrapitops.plan.extension.ExtensionServiceImplementation.updateServerValues(ExtensionServiceImplementation.java:196) ~[?:?]
at com.djrapitops.plan.extension.ExtensionServiceImplementation.updateServerValues(ExtensionServiceImplementation.java:196) ~[?:?]
at com.djrapitops.plan.extension.ExtensionServiceImplementation.updateServerValues(ExtensionServiceImplementation.java:196) ~[?:?]
at com.djrapitops.plan.extension.ExtensionServiceImplementation.updateServerValues(ExtensionServiceImplementation.java:196) ~[?:?]
at com.djrapitops.plan.extension.ExtensionServiceImplementation.lambda$register$0(ExtensionServiceImplementation.java:102) ~[?:?]
at com.djrapitops.plan.system.processing.Processing.lambda$submitNonCritical$0(Processing.java:73) ~[?:?]
at java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1590) ~[?:1.8.0_202]
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:1.8.0_202]
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:1.8.0_202]
at java.lang.Thread.run(Thread.java:748) [?:1.8.0_202]
Caused by: java.lang.reflect.InvocationTargetException
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_202]
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_202]
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_202]
at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_202]
at com.djrapitops.plan.extension.implementation.providers.MethodWrapper.callMethod(MethodWrapper.java:72) ~[?:?]
... 18 more
Caused by: java.lang.NullPointerException
at com.djrapitops.extension.DiscordSRVExtension.getLinkedAccountCount(DiscordSRVExtension.java:187) ~[?:?]
at com.djrapitops.extension.DiscordSRVExtension.accountsLinkedPerGuild(DiscordSRVExtension.java:217) ~[?:?]
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_202]
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_202]
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_202]
at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_202]
at com.djrapitops.plan.extension.implementation.providers.MethodWrapper.callMethod(MethodWrapper.java:72) ~[?:?]
... 18 more
```

**Comment by Vankka** [comment]
This issue is now fixed in Plan v4.8.3, https://github.com/plan-player-analytics/Plan/releases/tag/4.8.3
